app/services/organization_templates.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class OrganizationTemplates
+  def self.all
+    template_files = Dir.glob("lib/templates/*.yml")
+
+    templates = []
+    template_files.each do |file|
+      content = YAML.load_file(file)
+      templates << { "name" => content["name"], "id" => content["id"] }
+    end
+
+    templates
```

Comment:
```suggestion
    @all ||= Dir.glob("lib/templates/*.yml").map do |file|
      content = YAML.load_file(file)
      { "name" => content["name"], "id" => content["id"] }
    end
```
Note that this can be simplified with the benefit of [memoizing](https://www.justinweiss.com/articles/4-simple-memoization-patterns-in-ruby-and-one-gem/) the result

Reply:
Done